index.html

```diff
@@ -184,7 +184,7 @@ <h2>About Myself</h2>
                     don’t hesitate to contact me.
                 </p>
                 <p class="about-connect-text">LET'S CONNECT</p>
```

Comment:
- [ ] [Optional] Well, your social icons titles look visually good. However, instead of using a `<div>` tag, it would be more semantically appropriate to use a heading tag, such as `<h3>` or `<h4>`, to represent the title. This will improve the structure and accessibility of your code. Please consider updating it accordingly. Thank you!